tests/flow/test_mst.py

```
@@ -0,0 +1,477 @@
+from common import *
```

Ruff (0.11.9)

`from common import *` used; unable to detect undefined names

(F403)